app/api/search/specs/search.spec.js

```diff
@@ -375,6 +376,34 @@ describe('search', () => {
       expect(both.rows.length).toBe(2);
     });
 
+    it('should filter by range values using descriptive timestamps', async () => {
+      spyOn(date, 'descriptionToTimestamp').and.callFake(value => {
+        if (value === 'first-day-last-month') {
+          return 15;
+        }
+        if (value === 'last-day-last-month') {
+          return 25;
+        }
+        return 'not-catched';
+      });
+
+      try {
+        const results = await search.search(
+          {
+            types: [ids.template1],
+            filters: {
+              relationshipdate: { from: 'first-day-last-month', to: 'last-day-last-month' },
+            },
+          },
+          'en'
+        );
+
+        expect(results.rows[0].title).toBe('Inherited 1 EN');
+      } catch (err) {
+        fail(err);
+      }
```

Comment:
@RafaPolit try catch block here is not needed, if something fails jest will report the error properly.

Reply:
No, it didn't, it was not reporting the error returned by the await.  It was "just" failing.  Will look again.

Reply:
I have created a tech debt with this problem and with a quick explanation:
https://github.com/huridocs/uwazi/issues/4193